syntax = 'proto3';

import 'hoj_judge/protos/subtask.proto';

message SubtaskResponse {
  Subtask subtask = 1;
  uint32 verdict = 2;
  uint32 time_used = 3;
  uint32 mem_used = 4;
}
